src/locales/en.json

```diff
@@ -170,20 +171,20 @@
     "loud_voice": "Loud talking (shouting, talking over music, singing)",
     "person": {
       "average": "An average person in your area",
-      "frontline": "An essential or front-line worker in your area",
-      "nonFrontline": "Someone in your area who is NOT an essential or front-line worker",
+      "frontline": "A healthcare or social worker",
+      "nonFrontline": "A random person in your area who works from home",
```

Comment:
Remind me: Why are we going with "random" instead of "average"?

I can see benefits to both. Just curious.

Also, seems like we should be consistent and use the same thing for both.

Reply:
shrugs. Changed to average.

I think "average" is a dumb word because literally no one is average.